zep_python/message/client.py

```diff
@@ -36,6 +38,35 @@ def __init__(self, aclient: httpx.AsyncClient, client: httpx.Client) -> None:
         self.aclient = aclient
         self.client = client
 
+    def _validate_cursor_limit(
+        self, limit: int, cursor: int
+    ) -> Optional[Dict[str, int]]:
+        """
+        Validates the cursor and limit parameters.
+
+        Parameters
+        ----------
+        limit : int
+            The number of messages to return per page.
+        cursor : int
+            The page number to return.
+
+        Raises
+        ------
+        ValueError
+            If the limit or cursor is invalid.
+        """
+        if limit is not None and cursor is not None:
+            if (
+                not isinstance(limit, int)
```

Comment:
Does the function signature not check the type for limit and cursor?

Reply:
If so, we should just be able to do:

```
if limit <= 0 or cursor <= 0:
        raise ValueError("Both limit and cursor must be positive integers")

    return {"limit": limit, "cursor": cursor}
    ```

Reply:
thanks, that is weird! can't recall why I did that.